.github/workflows/build.yml

```diff
@@ -19,6 +19,10 @@ jobs:
         pip install -r requirements.txt -r requirements-dev.txt
     - name: Run pre-commit
       uses: pre-commit/action@v2.0.0
+    - name: Check for Sphinx doc warnings
+      run: |
+        cd docs
+        make html SPHINXOPTS="-W --keep-going"
```

Comment:
This isn't a blocker on merging, but any particular reason for not making this strict? In my experience people writing docs will make lots of mistakes, and the only way to catch these is to have strict checking on and to verify it as part of CI. Otherwise the docs get cruddy pretty quickly when less experienced developers start contributing.

Reply:
We could enable "nit-picky mode" (`-n`). I tried doing this, but it generated some errors that will need a bit more work to fix. I've opened https://github.com/pystatgen/sgkit/issues/86